pages/editor.vue

```diff
@@ -320,8 +320,7 @@
 
     .layer-background {
         padding-top: $spacer-5;
-        width: 100vw;
-        height: 100vh;
+        height: 100svh;
```

Comment:
I am not sure if `svh` is the right unit. I think in this `dvh` or `lvh` would be better. I think with svh there could be some edge case were we could run into the user being in a large view on mobile and the combination of 100svh and the heading not fully covering the screen, so that we have a bar at the bottom.